cms/server/admin/templates/participation_preview.html

```diff
@@ -0,0 +1,53 @@
+{% extends base.html %}
+
+{% block core %}
+
+<h1>Import users</h1>
+
+<div id="something">
+  <form enctype="multipart/form-data" action="{{ url_root }}/contest/{{ contest.id }}/users/import" method="POST">
+    <input type="hidden" name="action" value="save">
+    <table class="bordered">
+      <thead>
+        <tr>
+          <th>New</th>
```

Comment:
I added it to participation_preview, because rest of the columns weren't indicating it as clearly as user preview. Added "new" column to user import.